tests/sentry/api/validators/sentry_apps/test_select.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import absolute_import
+
+from sentry.testutils import TestCase
+
+from .util import invalid_schema, validate_component
+
+
+class TestSelectSchemaValidation(TestCase):
+    def setUp(self):
+        self.schema = {
+            'type': 'select',
+            'name': 'title',
+            'label': 'Title',
+            'options': [
+                ['Stuff', 'stuff'],
+                ['Things', 'things'],
+            ]
+        }
+
+    def test_valid_schema_with_options(self):
+        validate_component(self.schema)
+
+    def test_valid_schema_options_with_numeric_value(self):
+        self.schema['options'][0][1] = 1
+        self.schema['options'][1][1] = 2
+
+        validate_component(self.schema)
+
+    def test_valid_schema_with_uri(self):
```

Comment:
would we also want to test an invalid uri format?

Reply:
We might want to look into using a regex to validate them in the future – I'm not sure the `format: uri` really does a whole lot currently. I think it's to flesh that out later.